communication.rst: Add a link to the forum group list (#571)
ansible/ansible-documentation: docs/docsite/rst/community/communication.rst

The `Ansible Community Forum <https://forum.ansible.com>`_ is a single starting point for questions and help, development discussions, events, and much more. `Register <https://forum.ansible.com/signup?>`_ to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!

> Take a look at the `forum groups <https://forum.ansible.com/g>`_ and join ones that match your interests. Many of the groups will automatically subscribe you to related posts.

Reviewer: ```suggestion
Take a look at the `forum groups <https://forum.ansible.com/g>`_ and join ones that match your interests.
In most cases joining a forum group automatically subscribes you to related posts.
```

Author: merged, thanks, PTAL

The `Ansible Community Forum <https://forum.ansible.com>`_ is a single starting point for questions and help, development discussions, events, and much more. `Register <https://forum.ansible.com/signup?>`_ to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!

Take a look at the `forum groups <https://forum.ansible.com/g>`_ and join ones that match your interests.

> In most cases joining a forum group automatically subscribes you to related posts.

Reviewer 2: ```suggestion
In most cases, joining a forum group automatically subscribes you to related posts.
```
